src/tests/ftest/launch.py

```diff
@@ -3164,6 +3176,10 @@ def main():
             args.logs_threshold = DEFAULT_LOGS_THRESHOLD
         args.slurm_setup = True
         args.user_create = True
+    elif args.mode == "gcp":
```

Comment:
Should this be more generic?
```suggestion
    elif args.mode not in ["normal", "manual"]:
```

Reply:
I'm okay with that